lib/platform/github/gh-graphql-wrapper.ts

```diff
@@ -0,0 +1,72 @@
+import { api } from './gh-got-wrapper';
+import { logger } from '../../logger';
+
+const gqlOpts = {
+  headers: {
+    accept: 'application/vnd.github.merge-info-preview+json',
+  },
+  json: false,
```

Comment:
Why this? Response is json

Reply:
Working on it, didn't expected the problem goes so deep